content/docs/2.0/operate.md

```diff
@@ -0,0 +1,14 @@
++++
+title = "Operate"
++++
+
+## Prometheus Exporter Metrics
+
+The KEDA Metrics Adapter exposes Prometheus metrics which can be scraped on port `9022` (this can be changed by setting the `metrics-port` argument for the Metrics Adapter) at `/metrics`.  The metrics collected in the Metrics Adapter are only active when the HPA is active (> 0 replicas).
+
+The following metrics are being gathered:
+
+- `keda_metrics_adapter_scaler_error_totals` - The total number of errors encountered for all scalers
+- `keda_metrics_adapter_scaled_object_error_totals`- The number of errors that have occurred for each scaled object
```

Comment:
Given ScaledJobs are now seperate, should we report the metric separately?

Reply:
I believe all the scaling metrics for the scaled jobs is completely done inside the Operator and not the Metrics Adapter. Maybe the next step is add a prometheus exporter for the operator as well?

Reply:
+1 operator-sdk (which is using kubebuilder under the hood) has built in Prometheus Server and provides some basic metrics. It should be possible to extend the exposed metrics https://book.kubebuilder.io/reference/metrics.html#publishing-additional-metrics
But I'd keep it for a separate PR

Reply:
Ok, resolving!